post-types/datavizs.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+if (!class_exists('Odm_DataViz_Post_Type')) {
+
+    class Odm_DataViz_Post_Type
+    {
+        public function __construct()
+        {
+          add_action('init', array($this, 'register_post_type'));
+          add_action('save_post', array($this, 'save_post_data'));
+          add_filter('single_template', array($this, 'get_dashboards_template'));
+        }
+
+        public function get_dashboards_template($single_template)
+        {
+            global $post;
+
+            if ($post->post_type == 'dataviz') {
+                if (file_exists(plugin_dir_path(__FILE__).'templates/single-dataviz-'.$post->post_name.'.php')) {
```

Comment:
Ahhh .. Sorry i got confuse this one with dashboard post type. I think we don't need that one.